custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -175,6 +175,32 @@ class ArgumentsOptions:
     PUSH = "push"
 
 
+class ArgumentOptionsEnvVars:
+    TARGET_TYPE = "TARGET_TYPE"
+    CODE_DIR = "CODE_DIR"
+    NEGATIVE_CLASS_LABEL = "NEGATIVE_CLASS_LABEL"
+    POSITIVE_CLASS_LABEL = "POSITIVE_CLASS_LABEL"
+    CLASS_LABELS_FILE = "CLASS_LABELS_FILE"
+    ADDRESS = "ADDRESS"
+    MAX_WORKERS = "MAX_WORKERS"
+
+    MONITOR = "MONITOR"
+    WITH_ERROR_SERVER = "WITH_ERROR_SERVER"
+    SHOW_STACKTRACE = "SHOW_STACKTRACE"
+    PRODUCTION = "PRODUCTION"
+
+    VARS_WITH_VALUES = [
+        TARGET_TYPE,
+        CODE_DIR,
+        NEGATIVE_CLASS_LABEL,
+        POSITIVE_CLASS_LABEL,
+        CLASS_LABELS_FILE,
+        ADDRESS,
+        MAX_WORKERS,
+    ]
+    BOOL_VALUES = [WITH_ERROR_SERVER, SHOW_STACKTRACE, PRODUCTION, MONITOR]
```

Comment:
nit
VARS_WITH_VALUES -> VALUE_VARS
BOOL_VALUES           -> BOOL_VARS

Reply:
ok!

Reply:
(it's just the first one is called VARS_... and the second one is called ..._VALUES)

Reply:
the suggested renaming makes the naming a bit more consistent imo

Reply:
no problem!